src/ParameterTypes/passive_ren.jl

```diff
@@ -68,12 +74,13 @@ trainable(m::PassiveRENParams) = (direct = m.direct, )
 function direct_to_explicit(ps::PassiveRENParams{T}, return_h=false) where T
 
     # System sizes
-    nu = ps.nu
+    # nu = ps.nu
```

Comment:
@MrstupidJ why is this commented out? If `nu` is not needed, feel free to just delete it.